front/app/components/admin/FormResults/FormResultsQuestion/TextQuestion/Analysis/utils.ts

```diff
@@ -10,20 +9,6 @@ import {
   getYearFilter,
 } from 'containers/Admin/communityMonitor/components/LiveMonitor/components/HealthScoreWidget/utils';
 
-// Convert all values in the filters object to strings
-// This is necessary because the filters are passed as query params
-export const convertFilterValuesToString = (filters?: IInputsFilterParams) => {
-  return (
-    filters &&
-    Object.entries(filters).reduce((acc, [key, value]) => {
-      return {
-        ...acc,
-        [key]: JSON.stringify(value),
-      };
-    }, {})
-  );
-};
-
```

Comment:
The issue was that it converts strings to strings with quotes within the string. e.g. an object `{ data_from: '2025-07-02' }` becomes `{ data_from: '"2025-07-02"' }`, which is an invalid date value.

@IvaKop I couldn't come up with any reason to do this conversion, so I just removed it. But maybe I'm missing something?